CMakeLists.txt

```diff
@@ -95,3 +95,7 @@ add_library(${PROJECT_NAME}
 target_link_libraries(${PROJECT_NAME} re -lpthread -lm)
 set_target_properties(${PROJECT_NAME} PROPERTIES POSITION_INDEPENDENT_CODE ON)
 set_target_properties(${PROJECT_NAME} PROPERTIES SOVERSION ${PROJECT_SOVERSION})
+
+file(GLOB_RECURSE HEADERS include/*.h)
```

Comment:
Using glob is a [antipattern](https://cliutils.gitlab.io/modern-cmake/chapters/intro/dodonot.html). We should be explicit like in libre:

```cmake
set(HEADERS
include/rem_aac.h
include/rem_aubuf.h
include/rem_auconv.h
include/rem_audio.h
include/rem_aufile.h
include/rem_auframe.h
include/rem_au.h
include/rem_aulevel.h
include/rem_aumix.h
include/rem_auresamp.h
include/rem_autone.h
include/rem_avc.h
include/rem_dsp.h
include/rem_dtmf.h
include/rem_fir.h
include/rem_flv.h
include/rem_g711.h
include/rem_goertzel.h
include/rem.h
include/rem_vidconv.h
include/rem_video.h
include/rem_vid.h
include/rem_vidmix.h
)
```

Reply:
Used the same approach as for `re` library